specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -788,6 +950,109 @@
+      "properties": {
+        "allowVirtualNetworkAccess": {
+          "type": "boolean",
+          "description": "Whether the VMs in the local virtual network space would be able to access the VMs in remote virtual network space."
+        },
+        "allowForwardedTraffic": {
+          "type": "boolean",
+          "description": "Whether the forwarded traffic from the VMs in the local virtual network will be allowed/disallowed in remote virtual network."
+        },
+        "allowGatewayTransit": {
+          "type": "boolean",
+          "description": "If gateway links can be used in remote virtual networking to link to this virtual network."
+        },
+        "useRemoteGateways": {
+          "type": "boolean",
+          "description": "If remote gateways can be used on this virtual network. If the flag is set to true, and allowGatewayTransit on remote peering is also true, virtual network will use gateways of remote virtual network for transit. Only one peering can have this flag set to true. This flag cannot be set if virtual network already has a gateway."
+        },
+        "databricksVirtualNetwork": {
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "The Id of the databricks virtual network."
+            }
+          },
+          "description": " The remote virtual network should be in the same region. See here to learn more (https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/vnet-peering)."
+        },
+        "databricksAddressSpace": {
+          "$ref": "#/definitions/AddressSpace",
+          "description": "The reference to the databricks virtual network address space."
+        },
+        "remoteVirtualNetwork": {
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "The Id of the remote virtual network."
+            }
+          },
+          "description": " The remote virtual network should be in the same region. See here to learn more (https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/vnet-peering)."
+        },
+        "remoteAddressSpace": {
+          "$ref": "#/definitions/AddressSpace",
+          "description": "The reference to the remote virtual network address space."
+        },
+        "peeringState": {
+          "type": "string",
+          "description": "The status of the virtual network peering.",
+          "enum": [
+            "Initiated",
+            "Connected",
+            "Disconnected"
+          ],
+          "x-ms-enum": {
+            "name": "VirtualNetworkPeeringState",
+            "modelAsString": true
+          }
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/PeeringProvisioningState",
+          "description": "The provisioning state of the virtual network peering resource."
+        }
```

Comment:
provisioningState indicates the PUT is most likely async.  Is that not the case?

Reply:
No it is not async request.